packages/flutter/test/material/app_bar_theme_test.dart

```diff
@@ -50,14 +50,14 @@ void main() {
 
     final Material widget = _getAppBarMaterial(tester);
     final IconTheme iconTheme = _getAppBarIconTheme(tester);
-    final IconTheme actionsIconTheme = _getAppBarActionsIconTheme(tester);
+    final RichText actionIconText = _getAppBarIconRichText(tester);
     final DefaultTextStyle text = _getAppBarText(tester);
 
     expect(SystemChrome.latestStyle.statusBarBrightness, appBarTheme.brightness);
     expect(widget.color, appBarTheme.color);
     expect(widget.elevation, appBarTheme.elevation);
     expect(iconTheme.data, appBarTheme.iconTheme);
-    expect(actionsIconTheme.data, appBarTheme.actionsIconTheme);
+    expect(actionIconText.text.style.color, appBarTheme.actionsIconTheme.color);
```

Comment:
You don't have to get rid of checking the theme data. Check both.

Reply:
Oops, you're right. Done